frontend/src/metabase/visualizations/components/settings/ChartSettingFieldPicker.styled.tsx

```diff
@@ -2,13 +2,21 @@ import styled from "@emotion/styled";
 import { color } from "metabase/lib/colors";
 import Icon from "metabase/components/Icon";
 import SelectButton from "metabase/core/components/SelectButton";
+import Triggerable from "metabase/components/Triggerable";
 
 export const ChartSettingFieldPickerRoot = styled.div`
   display: flex;
   align-items: center;
   border: 1px solid ${color("border")};
   border-radius: 0.5rem;
   padding-right: 1rem;
+  overflow-x: hidden;
+  width: 100%;
```

Comment:
I'm not sure if this is needed, I tried removing these and it still works fine. Also, this div seems to target the whole selector that also includes the option (ellipsis) button. If I'm not mistaken you'd only set the overflow hidden to where you want to text to cut off, and you've already set overflow to hidden on the button content and that should be it.

![image](https://user-images.githubusercontent.com/1937582/188821258-ef788da4-4afd-480b-b857-081abdace2ee.png)

P.S. I'm no CSS expert, I just play around with adding/removing properties and see if things break apart or not lol. I just felt like these 2 are excessive.